### PR TITLE
Minor cleanup of SSL server methods, missing macros

### PR DESCRIPTION
**Pull request by earlephilhower**
The first of these commits removes duplicated code for loading certs.  There's already a method to do this, so the add'l ones added for support of ESP8266WiFiServerSecure were not needed.

The second one fixes a compile error found in the WebServerSecure class when HTTP_SERVER debugging is enabled.  The output macro wasn't brought in from the original, non-secure file.

### CODE REVIEW

**Comment by d-a-v** [comment]
#4274 collision :) I'll update mine

File: libraries/ESP8266WiFi/src/WiFiClientSecure.cpp [hunk 1]
```diff
@@ -471,24 +463,11 @@ WiFiClientSecure::WiFiClientSecure(ClientContext* client, bool usePMEM, const ui
...
-        uint8_t *stackData = (uint8_t*)alloca(max(certLen, rsakeyLen));
-        if (rsakey && rsakeyLen) {
-              memcpy_P(stackData, rsakey, rsakeyLen);
-              _ssl->loadServerRSAKey(stackData, rsakeyLen);
-        }
-        if (cert && certLen) {
-            memcpy_P(stackData, cert, certLen);
-            _ssl->loadServerX509Cert(stackData, certLen);
-        }
+        if (rsakey && rsakeyLen) _ssl->loadObject_P(SSL_OBJ_RSA_KEY, rsakey, rsakeyLen);
```
Comment (devyte) [inline libraries/ESP8266WiFi/src/WiFiClientSecure.cpp]: Please no 1-line if statements. Just put the _ssl line indented under the if.

**Comment by devyte** [comment]
My one comment is just about readability

File: libraries/ESP8266WiFi/src/WiFiClientSecure.cpp (hunk 1 above)
Comment (earlephilhower) [reply libraries/ESP8266WiFi/src/WiFiClientSecure.cpp]: My bad!  I try and cuddle all if{}s but sometimes one just slips by on me.  I've seen too many spots where folks have a 1-line, uncuddle'd if and decide "I'll put in a debug statement here" and end up changing the logic.

**Pull request merged**